includes/api/class-wc-admin-rest-products-controller.php

```diff
@@ -124,21 +124,24 @@ public static function add_wp_query_filter( $where, $wp_query ) {
 			$search = $wpdb->esc_like( $search );
 			$search = "'%" . $search . "%'";
 			$where .= " AND ({$wpdb->posts}.post_title LIKE {$search}";
-			$where .= wc_product_sku_enabled() ? ' OR ps_post_meta.meta_key = "_sku" AND ps_post_meta.meta_value LIKE ' . $search . ')' : ')';
+			$where .= wc_product_sku_enabled() ? ' OR wc_product_meta_lookup.sku LIKE ' . $search . ')' : ')';
```

Comment:
This will require updating the required WC version to 3.6 which we should probably do anyway. But, the admin notices & readme files will also need to be updated.

Reply:
>This will require updating the required WC version to 3.6

This is already the case